README.md

```diff
@@ -97,6 +97,21 @@ Execute the command with your venv (or virtualenv) environment.
 
 ### Common options
 
+#### Option: python
+
+By default, this tools finds the packages from the environment pip-licenses is launched from, by searching in current python's `sys.path` folders. In the case you want to search for packages in an other environment (e.g. if you want to run pip-licenses from its own isolated environment), you can specify a path to a python executable. The packages will be searched for in the given python's `sys.path`, free of pip-licenses dependencies.
```

Comment:
I also checked the importlib_metadata source code. It was using `sys.path` by default as you described 👍 

There is a ToC at the top of the README document. Could you also add a link to "Option: python"?